src/libcore/option.rs

```diff
@@ -269,6 +270,15 @@ impl<T> Option<T> {
         }
     }
 
+    /// Converts from `Option<T>` to `Option<PinMut<'_, T>>`
```

Comment:
I think this should say "from `PinMut<'_, Option<T>` to".

Reply:
If this is the case, we should better change the docs of `as_ref()` and `as_mut()` as well.